tests/dummy/app/templates/docs/faq/common-issues.md

```diff
@@ -1,37 +1,5 @@
-
-```js
-mainWindow = new BrowserWindow({
-    webPreferences: {
-        webSecurity: false,
-        allowRunningInsecureContent: false,
-    }
-});
-```
-
-This disables the same-origin policy, so this should done only after careful evaluation.
-
-If you need to read a local file, using the [`FileReader`](https://developer.mozilla.org/en-US/docs/Web/API/FileReader) API might be another good option that works without changing the `webPreferences`.
-
-
-## When I make AJAX requests they fail due to CORS problems (`Access-Control-Allow-Origin` header)
-
-This is also a side-effect of using `electron-protocol-serve`. One solution is to add an `Access-Control-Allow-Origin` header to the endpoint you are accessing, but that requires that you have control over that endpoint and that this fits with the security model of the server. Otherwise, this can be solved by setting `webPreferences` when opening your `BrowserWindow` as described above. However, this is only safe if you know you will never be running untrusted code in the `BrowserWindow` or allowing the `BrowserWindow` to navigate to a non-local URL.
-
-The reason this is necessary has to do with Electron's configuration of WebKit. Electron sets the [allow_universal_access_from_file_urls](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitSettings.html#WebKitSettings--allow-universal-access-from-file-urls), which disables the same-origin policy for Javascript loaded from `file:` URLs. However, since `electron-protocol-serve` causes us to load files from the custom `serve:` protocol, WebKit doesn't know that they are local files and enforces the same-origin policy by default, so we have to disable it using `webPreferences`.
-
-You should carefully consider if doing this is safe, as it potentially opens up new security holes that vanilla Electron using `file:` URLs does not. The difference is that the `allow_universal_access_from_file_urls` option evaluates the situation on each request, and if the current origin isn't a `file:` URL, it will enforce the same-origin policy. When using `webPreferences` you are disabling the same-origin policy for the entire window, so if the window somehow navigates to an `http:` URL, the same-origin policy will still be disabled. So be sure to consider your circumstances carefully from a security standpoint when doing this, and ensure that if your window needs to display remote or untrusted content, it does so in a sandboxed `WebView`.
-
-
 ## I can't open the Ember Inspector in the packaged app
 
 This is known and desired behavior of Electron. The Developer Tools are still accessible programmatically or via a shortcut (see below). However, the packages needed for [DevTron](https://github.com/electron/devtron) and the [Ember Inspector](https://github.com/emberjs/ember-inspector) addons are not currently bundled with production builds.
```

Comment:
Side question: weren't we talking about changing this behavior? Maybe I'm thinking of something else (can't find the issue I thought I remembered seeing....) Not that it would belong in this PR, but I can't remember if this info is still accurate.

Reply:
Yeah, it is still accurate, although changing it is super easy, and we could provide instructions for making them always accessible (just move the packages from `devDependencies` to `dependencies` and then get rid of the `isDev` check around them).

If you think it's worthwhile, feel free to file an issue.